model/fv_arrays.F90

```diff
@@ -875,23 +887,22 @@ module fv_arrays_mod
   real(kind=R_GRID) :: deglon_start = -30., deglon_stop = 30., &  !< boundaries of latlon patch
                        deglat_start = -30., deglat_stop = 30.
 
-   logical :: regional = .false.       !< Default setting for the regional domain.
-
-   integer :: bc_update_interval = 3   !< Default setting for interval (hours) between external regional BC data files.
-
-  integer :: nrows_blend = 0          !< # of blending rows in the outer integration domain.
-  logical :: write_restart_with_bcs = .false.   !< Default setting for using DA-updated BC files
-  logical :: regional_bcs_from_gsi = .false.    !< Default setting for writing restart files with boundary rows
-
-
   !>Convenience pointers
   integer, pointer :: grid_number
 
   !f1p
   logical  :: adj_mass_vmr = .false. !TER: This is to reproduce answers for verona patch.  This default can be changed
                                      !     to .true. in the next city release if desired
-  !integer, pointer :: test_case
-  !real,    pointer :: alpha
+
+  logical :: w_limiter = .true. ! Fix excessive w - momentum conserving --- sjl
+
+  ! options related to regional mode
+  logical :: regional = .false.       !< Default setting for the regional domain.
+  integer :: bc_update_interval = 3   !< Default setting for interval (hours) between external regional BC data files.
+  integer :: nrows_blend = 0          !< # of blending rows in the outer integration domain.
+  logical :: write_restart_with_bcs = .false.   !< Default setting for using DA-updated BC files
+  logical :: regional_bcs_from_gsi = .false.    !< Default setting for writing restart files with boundary rows.
+  logical :: pass_full_omega_to_physics_in_non_hydrostatic_mode = .false.  !< Default to passing local omega to physics in non-hydrostatic mode
```

Comment:
I see this in a few places. Where did this come from?

Reply:
That logica was brought into the development repo in Sept 2020 by Kai from github::dev/emc by Kai as part of commit [273f7afb](https://gitlab.gfdl.noaa.gov/fv3team/atmos_cubed_sphere/-/commit/273f7afb6523038a41ffbdc6c9a1e850388a15da)